src/getting_started/development/04_vscode_ext.md

```diff
@@ -0,0 +1,108 @@
+
+To install, find svlangserver in the VSCode marketplace. Then create and update a settings file at <WORKSPACE PATH>/.vscode/settings.json with the desired settings as detailed in the link above. The extension's commands and troubleshooting can also be found in the link.
+
+---
+
+### C++ Pack
+
+[This extension pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools-extension-pack) has 4 sets of popular extensions:
+  - [C/C++ support](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools)
+  - [C/C++ Themes](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools-themes)
+  - [CMake support](https://marketplace.visualstudio.com/items?itemName=twxs.cmake)
+  - [CMake Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cmake-tools)
+  
+---
+  
+### Doxygen Generator
+  
+[Doxygen](https://github.com/doxygen/doxygen) is a documentation generator that extracts documentation from C++ source code comments. This [Doxygen comment generator](https://marketplace.visualstudio.com/items?itemName=cschlosser.doxdocgen) creates Doxygen comments for a variety of code snippets: constructors, functions, file descriptions, etc.
+  
+---
+
+### Spell Checker
+
+This [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) does exactly that: it is a basic spell check that can be customized and has a variety of add-on language dictionaries.
+
+---
+
+## Git & GitHub
+
+### GitLens
+  
+[GitLens](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens) is a customizable extension that includes a wide variety of features such as a Git command palette, and detailed commit, repo, and branch histories. Full details can be found in the link above.
+
+### GitHub Actions
+
+[GitHub Actions](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-github-actions&ssr=false#overview) lets users manage workflows and view workflow history for github.com hosted repositories.
+
+### GitHub Pull Requests
+
+The [GitHub Pull Requests and Issues](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github) extensions lets you review and manage GitHub PRs and issues directly in VSCode.
+
+---
+
+## Markdown
+
+### Markdown All in One
+
+[Markdown All in One](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one) is a collection of Markdown commands and features like keyboard shortcuts, math, table of contents, etc.
+
+### Better Comments
+
+[Better Comments](https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments) helps to create more understandable, categorizable comments for your code, and works with C++ and Verilog.
```

Comment:
This is not a markdown extension, it can go around the doxygen one because both pretty much help with comments.